### ISSUE TITLE
Tags Entered Using One-Click Reblog Are Not Posting

### ISSUE THREAD

**Issue by blindcat97** [new-xkit/XKit#333]
I'm not sure if this issue is happening on my main blog, but I tag almost /every single post/ on my side blog, and not a single one is showing up when I actually post. I tag through the One-Click Reblog extension. This has been happening for over a week (not sure precisely how long, I make too many posts a day to look further back). 

In addition, if I hit escape after writing the tags and then press T to go back, as I have done for many months, the tags field is empty again. That part has been happening since I first upgraded to New XKit. In the old XKit, the tags field remained full after pressing escape and then returning. 

I'd like to add that I always press T, type my tags, escape, and then R to reblog, in that exact order constantly without fail. Is pressing escape the problem here? I'm going to do some additional testing; another post on that to follow.

**Comment by blindcat97** [comment]
Okay, some new information: 

Pressing escape and R after selecting that it post to a different blog than my default clears the option (meaning it still posts to my default). However, NOT pressing escape, but rather clicking on the reblog button, it posts the non-default blog I had selected. I was unable to test if tagging worked on my main blog while using escape R because of this issue.

On my side blog, clicking the one-click reblog button, rather than pressing escape R, preserved my tags. The post I tried to send to my main via escape R, that ended up posting to my default side blog, still did not have tags. 

This is all pretty useless information to anyone who uses their mouse regularly, but this is very important to anyone like me who primarily operates tumblr via their keyboard. 

One more post to follow as I test it on Firefox. 

VERSIONS OF STUFF
I'm running Windows 10 Pro, using Google Chrome 44 (specifically, Version 44.0.2403.155 m), XKit version 7.7.0.
